lib/network/kmip/transport/TransportTemplate.ts

```diff
@@ -90,9 +90,14 @@ export default class TransportTemplate {
         readyCallback: (error: Error | null) => void,
     ) {
         try {
+            const options = {
+                ...this.options.tls,
+                host: '127.0.0.1',
```

Comment:
this code is used in production, we should not set the host here (esp. it overrides `this.optins.tls`)
--> should be set in the tests i guess?